production/scrapers/lasd.R

```diff
@@ -48,12 +48,29 @@ lasd_pull <- function(x, wait = 5){
 lasd_restruct <- function(x){
     
     # Trim whitespace and brighten image 
-    x <- magick::image_trim(x) %>% 
+    x <- x %>%
+        #magick::image_read() %>%
+        magick::image_trim() %>% 
         magick::image_modulate(brightness = 120)
     
     w_ <- magick::image_info(x)$width
     h_ <- magick::image_info(x)$height
     
+    confirmed.crop <- "570x30+580+400"
+    recover.crop <- '570x30+580+710'
+    death.crop <- '570x30+580+780'
+    quarantine.crop <- "570x30+590+950"
+    drop.neg.asymp.1.crop <- "570x25+600+562"
+    drop.neg.asymp.2.crop <- "570x25+600+585"
+    drop.neg.symp.crop <- "570x25+0+562"
+    drop.pos.asymp.crop <- '570x30+580+502'
+    drop.pos.symp.crop <- "562x25+0+508"
```

Comment:
When I change the height to be 28 here, it works fine. EG `drop.pos.symp.crop <- "562x28+0+508"`

Reply:
Confused now reviewing this PR for changes - fine with changing the deaths number warning (I asked and we decided not to change this in a data meeting but I'm for it) and removing the hashed section. This original commit still works for me though and the comment isn't clear as to why it is setting new dimensions.